Problem: contributors in toml files are hard to maintain

theduke opened the pull request:
Every contributor is mentioned in a dedicated file anyway, so
maintaining a list of contributors in each toml file is rather
cumbersome and redundant.

Solution: don't mention contributors in Cargo.toml's.

yrashk commented:
I think `authors` is mandatory, isn't it? http://doc.crates.io/manifest.html

theduke commented:
Hmm oh yeah, mandatory for publishing. 

Maybe just keep you in there as the project starter?

DIesel and serde both have just one author in there.

yrashk commented:
I had a simple idea of writing a tiny script that we can run when we update CONTRIBUTORS that'll just propagate it to every Cargo.toml. This way everybody is properly mentioned in crates (when we get to publishing crates).

yrashk commented:
Either way works, really. Just copying and pasting manually (as I do right now) doesn't work  :)

theduke committed:
Problem: contributors in toml files are hard to maintain

Solution: just mention project initiator in toml,
everyone else is mentioned in CONTRIBUTORS file.

theduke commented:
That will grow quite tedious when there are more contributors though, in my opinion.

Just imagine having to scroll over 50 contributors before you get to the dependencies when you want to check out the crate. 

But it's up to you. 

I've pushed a version with removing everyone but you, but feel free to close it, I dont mind. 

Personally I'm more than satisfied with being mentioned in the CONTRIBUTORS file.

yrashk commented:
I have an idea. How about we use `authors` for active maintainers (essentially those with a few contributions who decided they want to have the extra burden of merging stuff as well :)?
